governance/drift-check.sh

```
@@ -684,6 +684,54 @@ else
+    drift "repo '$r' AGENTS.md 拉取失败（协议块对账跳过——fail-closed，ADR-0055 §17）"
+    PROTO_OK=1
+    continue
+  fi
+  BLOCK=$(proto_block <<<"$RAW")
+  if [[ -z "$BLOCK" ]]; then
+    drift "repo '$r' AGENTS.md 缺协议块标记（entry_protocol 已申报——宪法 §11 统一入口）"
+    PROTO_OK=1
+    continue
+  fi
+  if [[ "$BLOCK" != "$CANON_BLOCK" ]]; then
+    drift "repo '$r' 协议块与 template-service 不一致（首处差异: $(diff <<<"$CANON_BLOCK" <<<"$BLOCK" | head -3 | tr '\n' ' ' | cut -c1-160)）"
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stdin. Use cat, tee, or pass filenames instead.

(SC2261)